import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/CFindService.java

```diff
@@ -135,15 +135,13 @@ public void run() {
         MonitoringService.addEvent(Event.CFIND_CANCEL);
         as.tryWriteDimseRSP(pc, Commands.mkCFindRSP(cmd, Status.Cancel));
       } catch (IDicomWebClient.DicomWebException e) {
-        log.error("CFind qido-rs error", e);
+        log.error("CFind Qido-rs error", e);
         MonitoringService.addEvent(Event.CFIND_QIDORS_ERROR);
-        as.tryWriteDimseRSP(pc, Commands.mkCFindRSP(cmd, e.getStatus()), e.getAttributes());
+        sendErrorResponse(e.getStatus(), e.getMessage());
```

Comment:
We can remove the getAttributes() method on DicomWebException since there are no other callers.

Reply:
Ok